torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -142,6 +142,18 @@ int64_t _safe_size(IntArrayRef sizes, IntArrayRef dim) {
   return size;
 }
 
+c10::SymInt _safe_size(c10::SymIntArrayRef sizes, c10::IntArrayRef dim) {
+  c10::SymInt size = 1;
+  if (sizes.size() == 0) {
+    return 1;
```

Comment:
nit return size;

Reply:
fwiw this is symmetric with the original